src/pages/login.tsx

```diff
@@ -0,0 +1,43 @@
+import { type NextPage } from "next";
+import React from "react";
+import { Layout } from "../components/Layout";
+import { LoginForm } from "../components/LoginForm";
+import { useAuth } from "../hooks/useAuth";
+import { useToast } from "../hooks/useToast";
+import { AuthProvider } from "../providers/AuthProvider";
+import { LoginValues } from "../schemas/loginSchema";
+import { api } from "../utils/api";
+import Router from "next/router";
```

Comment:
Let's use `const router = useRouter();` instead of `import Router from "next/router";` I've never seen someone using `Router` object directly. Maybe it is okay. but let's go with a hook version